lib/seadsa/AllocWrapInfo.cc

```diff
@@ -87,7 +87,7 @@ void AllocWrapInfo::initialize(Module &M, Pass *P) const {
 
 void AllocWrapInfo::findAllocs(Module &M) const {
   for (Function &F : M) {
-    if (m_allocs.count(F.getName()))
+    if (m_allocs.count(F.getName().str()))
```

Comment:
why is `.str()` needed? Maybe we need to change the type of `m_allocs` instead?

Reply:
LLVM 11 removes the implicit cast from `llvm::StringRef` to `std::string` so I had to do the cast explicitly otherwise there would be a compilation error.

Reply:
With respect to changing the type `m_allocs`, I did change it to `llvm::StringRef` but our discussion suggested that it may be unsafe. So I changed it back.

Reply:
then maybe it is time to switch to stringref

Reply:
at that time you had an unexplained crash so I suggested to not do two changes at once.  Now that we understood the root cause for the crash we can go back to stringref

Reply:
Done.